swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+basePath: /api/1.0
+consumes:
+  - application/json
+host: sandbox.kuzzle.io:7511
+info:
+  contact:
+    email: hello@kuzzle.io
+    name: Kuzzle team
+    url: 'http://kuzzle.io'
+  description: The Kuzzle REST API
+  license:
+    name: Apache 2
+    url: 'http://opensource.org/licenses/apache2.0'
+  title: Kuzzle API
+  version: 1.0.0
+paths:
+  '/:index':
+    put:
+      description: 'Controller: admin. Action: createIndex.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection':
+    put:
+      description: 'Controller: write. Action: createCollection.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/:id':
```

Comment:
missing also methods for route '/:index/:collection/:id':
*  "get" => read/get
* "delete" => write/delete
* "delete" => security/deleteRole